src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js

```diff
@@ -399,7 +400,15 @@ const OncoGridWrapper = compose(
             <div style={{ flexGrow: 1 }} className="oncogrid-legend">
               {heatMapMode
                 ? <StepLegend rightLabel="More Mutations" />
-                : <SwatchLegend colorMap={colorMap} />}
+                : <SwatchLegend
+                    colorMap={Object.entries(colorMap).reduce(
+                      (acc, [key, val]) =>
+                        Object.assign(acc, {
+                          [key.replace("_variant", "")]: val
+                        }),
+                      {}
+                    )}
+                  />}
```

Comment:
kind of weird that the first param is value not key, seems unintuitive.

```
_.mapKeys({ 'a': 1, 'b': 2 }, function(value, key) {
  return key + value;
});
// => { 'a1': 1, 'b2': 2 }
```

https://lodash.com/docs/#mapKeys